commons/zenoh-protocol-core/src/lib.rs

```diff
@@ -461,99 +461,12 @@ pub mod queryable {
 
 /// The kind of consolidation.
 #[derive(Debug, Clone, PartialEq, Eq, Copy)]
-#[repr(u8)]
 pub enum ConsolidationMode {
     None,
     Lazy,
     Full,
 }
```

Comment:
I prefer the words `Monotonic` and `LastValue` (as Pierre mentioned on discord) to `Lazy` and `Full`. It is more explicit and clear to the user.

Reply:
Since the `LastValue` is based on `timestamp`, shouldn't be `LatestValue` to be more explicit about its relationship with ordered time?

Reply:
Agreed, just forgot to actually write it, thaks @sreeja :)

Reply:
Moreover, `Latest` would be simpler and more aligned with `None` and `Monotonic`.